File: DayOne/Python/book_suggestion_system.py
books = []

def add_book(title):
    if title in books:
        return False
    books.append(title)
    return True

def remove_book(title):
    if title in books:
        books.remove(title)
        return True
    return False

def update_book(old, new):
    if old in books:
        index = books.index(old)
        books[index] = new
        return True
    return False

def show_books():
    return books

File: DayOne/JavaScript/BookSuggestionSystem.js
let books = [];

function addBook(title) {
    if (books.includes(title)) {
        return false;
    }
    books.push(title);
    return true;
}

function removeBook(title) {
    const index = books.indexOf(title);
    if (index !== -1) {
    books.splice(index, 1);
    return true;
    }
    return false;
}

function updateBook(oldTitle, newTitle) {
    const index = books.indexOf(oldTitle);
    if (index !== -1) {
    books[index] = newTitle;
    return true;
    }
    return false;
}

function showBooks() {
    return books;
}

module.exports{addBook, removeBook, updateBook, showBooks};
